code/modules/power/floodlight.dm

```diff
@@ -0,0 +1,98 @@
+
+/obj/structure/floodlight_frame
+	name = "floodlight frame"
+	desc = "A bare metal frame looking vaguely like a floodlight. Requires wiring.
```

Comment:
you didn't close this string (this is why the file diff is all blue except strings, because all of the code is in a string!)